### PR TITLE
61 liquidity dependencies

### PR DESCRIPTION
**Pull request by blewater**
Strip the AccountKeeper dependency from the LiquidityProviderModule.
Add gPRC, cli query namespace with List(), Mintable() requests.

```sh
emd q liquidityprovider
Querying commands for the liquidity provider module

Usage:
  emd query liquidityprovider [flags]
  emd query liquidityprovider [command]

Aliases:
  liquidityprovider, lp

Available Commands:
  list        List liquidity providers
  mintable    List mintable coins

Flags:
  -h, --help   help for liquidityprovider

Global Flags:
      --chain-id string     The network chain ID
      --home string         directory for config and data (default "/home/mar/.emd")
      --log_format string   The logging format (json|plain) (default "plain")
      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
      --trace               print out full stack trace on errors

Use "emd query liquidityprovider [command] --help" for more information about a command.
```

### CODE REVIEW

**Review by haasted — APPROVED** [review]
Nice job!

File: x/issuer/keeper/keeper.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func NewKeeper(cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, lpk lp.Keeper,
 	}
 }
 
-func (k Keeper) IncreaseMintableAmountOfLiquidityProvider(ctx sdk.Context, liquidityProvider sdk.AccAddress, issuer sdk.AccAddress, mintableIncrease sdk.Coins) (*sdk.Result, error) {
+func (k Keeper) IncreaseMintableAmountOfLiquidityProvider(ctx sdk.Context, liquidityProvider string, issuer sdk.AccAddress, mintableIncrease sdk.Coins) (*sdk.Result, error) {
```
[the remark this answers is not in the document]
Comment (blewater) [reply x/issuer/keeper/keeper.go:41]: Updated API to use AccAddress.

**Comment by blewater** [comment]
Merged with SDK upgrade 0.42.6

File: x/issuer/keeper/keeper.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (haasted) [reply x/issuer/keeper/keeper.go:41]: Can't seem to find that commit. Did you push it?

[the remark this answers is not in the document]
Comment (blewater) [reply x/issuer/keeper/keeper.go:41]: Don't know why it's not showing here as it precedes the merge with the dev sdk upgrade. This is the commit https://github.com/e-money/em-ledger/commit/f74f051835ee1ff0aa191a75bf5a92bb8525dd07. You can also view it locally with `git log` after `gh pr checkout 83`

**Pull request merged**